src/main/scala/beam/router/r5/R5Wrapper.scala

```diff
@@ -1249,10 +1291,12 @@ class R5Wrapper(workerParams: R5Parameters, travelTime: TravelTime, travelTimeNo
           }
         } else 1f
 
-      (traversalTimeSeconds + (timeValueOfMoney * tollCalculator.calcTollByLinkId(
+      val fare: Double = traversalTimeSeconds / 60.0 * perMinuteCost + edge.getLengthM / METERS_IN_MILE * perMileCost
```

Comment:
I can't figure out a way to tell R5 to add a fixed cost to the itinerary when starting a RH access/egress leg, if anyone can figure that out it would be great to include.

I _think_ it won't make a big difference, though. R5 is already returning a bundle of options that should include each of RH->TRANSIT->RH, RH->TRANSIT->WALK, and WALK->TRANSIT->RH if they are available. Giving the per-mile and per-minute cost to R5 will let it find the true best alternative between each of these three categories, and within each of these categories there would be the same total baseCost. So as long as R5 isn't preemptively choosing between the different access/egress mode options I think it should be moot.

Reply:
Resolving this after confirming that it does work as expected -- R5 only uses this travelCostCalculator to determine the best routes given a sequence of legs (e.g. RH->TRANSIT->WALK or RH->TRANSIT->RH), so assuming that the base fare is constant geographically (which it is for now) it should work as intended